layouts/partials/footer.html

```diff
@@ -5,9 +5,12 @@
       <div class="row">
         <div class="col-12 col-lg-4 me-auto order-2 order-lg-1">
           <div class="footer-logo mt-7 mt-md-0 mb-5">
-            <a href={{relURL .logo_url}} class="">
+            <a href={{relURL .Site.Data.footer.logo_url}} class="first-logo-link">
               <img src={{relURL "images/chips-logo-white.svg"}} alt="logo" />
```

Comment:
While we are cleaning things up, it would be better if the alt text was "CHIPS Allaince Logo" and if the class did not specify location in it's name.